colibre/scripts/birth_velocity_dispersion_distribution.py

```diff
@@ -0,0 +1,97 @@
+"""
+Plots the birth velocity dispersion distribution.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+import unyt
+import traceback
+
+
+from swiftsimio import load
+from swiftpipeline.argumentparser import ScriptArgumentParser
+
+
+arguments = ScriptArgumentParser(
+    description="Creates a stellar birth velocity dispersion distribution plot, split by redshift"
+)
+
+snapshot_filenames = [
+    f"{directory}/{snapshot}"
+    for directory, snapshot in zip(arguments.directory_list, arguments.snapshot_list)
+]
+
+names = arguments.name_list
+output_path = arguments.output_directory
+
+plt.style.use(arguments.stylesheet_location)
+
+data = [load(snapshot_filename) for snapshot_filename in snapshot_filenames]
+number_of_bins = 256
+
+birth_velocity_dispersion_bins = unyt.unyt_array(
+    np.logspace(1.0, 3.5, number_of_bins), units="km**2/s**2"
```

Comment:
Let's use 

```suggestion
    np.logspace(0.25, 5.25, number_of_bins), units="km**2/s**2"
```

Based on my tests, some runs will prefer larges values than the current range, e.g.

![birth_velocity_dispersion_distribution](https://user-images.githubusercontent.com/20153933/202929517-5955b8f1-f6df-418d-9264-b5a36931bd47.png)